app/assets/stylesheets/_style.css.scss

```diff
@@ -9,8 +9,7 @@ nav > .inner,
 }
 
 nav {
-  $_nav_height: 50px;
-  height: $_nav_height;
+  height: $nav_height;
```

Comment:
Name of variable `nav_height` should be written in all lowercase letters with hyphens instead of underscores<br>Properties should be ordered background-color, color, height, padding